app/views/collections/_filters.html.erb

```diff
@@ -27,44 +27,44 @@
       </dd>
 
       <dt class="autocomplete search">
-        <%= label_tag "collection_filters_title",
-            ts("Filter by title or name") %>
+        <%= label_tag "title",
+            ts("Filter by title") %>
       </dt>
       <dd class="autocomplete search">
-        <%= text_field_tag "collection_filters[title]",
-            params[:collection_filters][:title],
+        <%= text_field_tag "title",
```

Comment:
Instead of using `_tag` helpers, you can rewrite this view to use `form_for` with the new CollectionSearchForm class. You can reference [the work filters view](https://github.com/otwcode/otwarchive/blob/f057b6b9f85ccdd26dcff2aca2efa17cf7f38e26/app/views/works/_filters.html.erb#L1-L7) for example.